spec/validation/message_spec.rb

```diff
@@ -1,29 +1,32 @@
 # frozen_string_literal: true
 require "spec_helper"
 
-module HathiTrust::Validation
+module HathiTrust::Validator
   describe Message do
     let(:args) do
       {
-        validation: "first_validation",
+        validator: HathiTrust::Validator::Base,
+        validation_type: :something,
         human_message: "test fail",
         level: Message::ERROR,
         extras: { a: 1, b: 2 }
       }
     end
 
-    describe "#validation" do
+    describe "#validation_type" do
       it "accepts a string" do
-        message = described_class.new(args.merge(validation: "val"))
-        expect(message.validation).to eql(:val)
-      end
-      it "accepts a class" do
-        message = described_class.new(args.merge(validation: Integer))
-        expect(message.validation).to eql(:Integer)
+        message = described_class.new(args.merge(validation_type: "val"))
+        expect(message.validation_type).to eql(:val)
       end
       it "accepts a symbol" do
-        message = described_class.new(args.merge(validation: :some_sym))
-        expect(message.validation).to eql(:some_sym)
+        message = described_class.new(args.merge(validation_type: :some_sym))
+        expect(message.validation_type).to eql(:some_sym)
+      end
+    end
+    describe "#validator" do
+      it "accepts a class" do
+        message = described_class.new(args.merge(validator: Integer))
+        expect(message.validator).to eql(Integer)
```

Comment:
Makes sense, I think this was inherited from @malakai97's original spec so out of scope for this PR - can you extract to another issue?